public/styles/login.scss

```diff
@@ -0,0 +1,250 @@
+%style-button {
```

Comment:
Most of these styling is similar to the signup style. Why not extend that?

Reply:
Feedback well received

Reply:
Has this been done?